src/components/UserActions/Strings.ts

```diff
@@ -0,0 +1,25 @@
+import { Translation, Language } from '../../utils/translation';
+
+Translation.register(Language.English, {
+    UserActions_no_actions: 'No actions available for this user',
+
+    QueriesList_more: 'Show More',
+    QueriesList_less: 'Show Less',
+    QueriesList_no_queries: 'No queries made by this user',
+
+    ClickedDocumentsList_more: 'Show More',
+    ClickedDocumentsList_less: 'Show Less',
+    ClickedDocumentsList_no_clicked_documents: 'No document clicked by this user',
+
+    UserActivity_start_date: 'Start Date',
+    UserActivity_start_time: 'Start Time',
+    UserActivity_duration: 'Duration',
+    UserActivity_other_event: 'Other Event',
+    UserActivity_other_events: 'Other Events',
+
+    UserActivity_search: 'User Query',
+    UserActivity_query: 'Query',
+    UserActivity_click: 'Clicked Document',
+    UserActivity_view: 'Page View',
+    UserActivity_custom: 'Custom'
+});
```

Comment:
@stremblay2 Toutes les string de ce fichier

Reply:
Where will we see this? "Custom" ?

Reply:
"Custom Action"